docs/4. Product Features/03. Flows/2. Flow Actions/12. Link Google Sheets.md

```
@@ -68,6 +68,14 @@ ___
 
 - Dynamic Menu: Read items from a sheet and send them as interactive list options.
 
+### Google Sheet Sync
+
+- When content to a linked google sheet in read mode is updated, the sheet needs to be `synced` using the sync button corresponding to the google sheet.
+- This ensures that the latest modifications made in the google sheet are avaiable to use in the Glific flows where the sheet is being used. 
+- Google sheet in read-mode and on auto-sync functionality sync every 24 hours. The google sync operations happen between 5-6am IST
+- To handle use case which demand that the google sheets be synced more than once a day or as per a custom logic, read more [here](https://docs.google.com/document/d/1Hix2ShKMJqfhOHrTc4GAT2JWc3ThyuZ8RvdO61wc2d4/edit?usp=sharing)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)